docs/process.rst

```diff
@@ -281,8 +281,8 @@ Utopic) will run into problems with running the functional tests.
     ``bzr merge ../trusty-release``.
   - If required, commit the changes by running the following:
     ``bzr ci -m "Merged changes from the released charm."``.
-  - Copy the new GUI release to the releases directory of the charm
-    (i.e. ``develop-trunk/releases``).
+  - If a new GUI tarball is actually available, copy the new GUI release to the
```

Comment:
When we are following those instructions to release the charm itself, not the GUI and the corresponding new charm.